contents/docs/surveys/branching-logic.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Branching logic
+sidebar: Docs
+showTitle: true
+availability:
+  free: none
+  selfServe: full
+  enterprise: full
+---
+
+It is useful to be able to display survey questions conditionally. For example, you may want to ask certain questions based on the responses received. Branching logic allows you to do this.
+
+## Simple branching
```

Comment:
Since `After this question, go to:` is also in the form creation, can you also add a heading for it in /docs/surveys/creating-surveys? You can link to this doc from there